package.json

```diff
@@ -10,15 +10,15 @@
     "access": "public"
   },
   "sideEffects": false,
-  "main": "dist/index.js",
+  "main": "dist/index.cjs.js",
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "files": [
     "dist/"
   ],
   "homepage": "https://objective-redux.github.io/Objective-Redux/",
   "scripts": {
-    "build": "npm run transpile && cd examples/react-redux && npm i && cd ../lazy-loading && npm i && cd ../server-side-rendering && npm i",
```

Comment:
should probably have a `build-examples` script or something? Didn't find it really necessary to build all the examples every time we want to build the package.

I think NPM/yarn workspaces could be created for these examples as well

Reply:
Yeah, this command is primarily for the GitHub Actions to be able to run the end-to-end tests. Agreed, though, this could use some refactoring.